app/main/views.py

```diff
@@ -157,17 +158,18 @@ def change_email():
 
     if emailForm.validate_on_submit():
         user = User.query.filter_by(email=emailForm.email.data).first()
-        if emailForm.confirmEmail.data == emailForm.email.data and user is None:
+        if user is None:
             current_user.email = emailForm.email.data
+            confirm_url = get_confirm_url(user)
+            send_conf_email(user, confirm_url)
```

Comment:
```suggestion
            confirm_url = get_confirm_url(current_user)
            send_conf_email(current_user, confirm_url)
```
`if user is None`, why would you be able to send an email?

Reply:
This is the current email the user has prior to change. So the email should not be existing in our database.

Reply:
Yes, so who exactly would `send_conf_email` be sending the email to? `None` is not a `user`.